vendors/circom/README.md

```diff
@@ -81,3 +81,11 @@ Optional arguments:
 --disable_fast_twiddles_mode
                     Disables fast twiddle mode on Icicle NTT domain initialization.
 ```
+
+## How to compile witness generator
+
+```shell
+bazel build //circomlib/build/...
+bazel-bin/circomlib/build/compile_witness_generator <full_path_to_large_cpp>
```

Comment:
It sounds like it can accept arbitrary huge cpp, please make it clear!

```suggestion
bazel-bin/circomlib/build/compile_witness_generator /path/to/main.cpp
```

Reply:
Please add option about fields!